app/javascript/components/conditions/RejectionModal/component.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import PropTypes from 'prop-types';
+import { inject, observer } from 'mobx-react';
+
+@inject('conditionSetStore')
+@observer
+class RejectionModal extends React.Component {
+  static propTypes = {
+    conditionSetStore: PropTypes.object,
```

Comment:
I'm not sure you'll need the store in this component. The only piece of data you need to share out is whether or not there are any translations set, right? So you could have that be a piece of state on this component that gets read by the link component.